src/libstd/num/f32.rs

```diff
@@ -680,6 +732,21 @@ impl f32 {
         unsafe { cmath::fminf(self, other) }
     }
 
+    /// Returns the remainder when `x` is divided by `y`
+    ///
+    /// ```
+    /// let x = 5.0f32;
+    /// let y = 2.0f32;
+    ///
+    /// assert_eq!(x.rem(y), 1.0);
+    /// ```
+    #[unstable(feature = "std_misc",
+                reason = "unsure about its place in the world")]
+    #[inline]
+    pub fn rem(self, other: f32) -> f32 {
```

Comment:
I believe that this is already exposed through the `Rem` trait (LLVM lowers `%` operations to calls to this function I believe).